csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -1779,95 +1914,104 @@ CSAF documents are based on JSON, thus the security considerations of [RFC8259]
 
 * **CSAF document**: A security advisory text document in the format defined by this document.
 * **CSAF producer**: A program which emits output in the CSAF format.
 * **Direct producer**: An analysis tool which acts as a CSAF producer.
 * **Converter**: A CSAF producer that transforms the output of an analysis tool from its native output format into the CSAF format.
 * **CVRF CSAF converter**: A CSAF producer which takes a CVRF document as input and converts it into a vaild CSAF document.
 * **CSAF content management system**: A program that is able to create, review and manage CSAF documents and is able to preview their details as required by CSAF viewer.
-* **CSAF post-processor**: A CSAF producer that transforms an existing CSAF document into a new CSAF document, for example, by removing or redacting security-sensitive elements.
+* **CSAF post-processor**: A CSAF producer that transforms an existing CSAF document into a new CSAF document, for example, by removing or redacting elements according to sharing policies.
 * **CSAF modifier**: A CSAF post-processor which takes a CSAF document as input and modifies the structure or values of properties. The output is a valid CSAF document.
 * **CSAF translator**: A CSAF post-processor which takes a CSAF document as input and translates values of properties into another language. The output is a valid CSAF document.
 * **CSAF consumer**: A program that reads and interprets a CSAF document.
 * **CSAF viewer**: A CSAF consumer that reads a CSAF document, displays a list of the results it contains, and allows an end user to view each result in the context of the artifact in which it occurs.
 * **CSAF management system**: A program that is able to manage CSAF documents and is able to display their details as required by CSAF viewer.
-* **CSAF asset matching system**: A program that connects to or is asset database and is able to manage CSAF documents as required by CSAF management system as well as matching them to assets of the asset database.
+* **CSAF asset matching system**: A program that connects to or is an asset database and is able to manage CSAF documents as required by CSAF management system as well as matching them to assets of the asset database.
+
+### 5.1.1 Conformance Clause 1: CSAF document
+
+A text file satisfies the "CSAF document" conformance profile if the text file:
 
-## 5.2 Conformance Clause 1: CSAF document
+* conforms to the syntax and semantics defined in section 3.
 
-A text file satisfies the "CSAF document" conformance profile if:
+### 5.1.2 Conformance Clause 2: CSAF producer
 
-* It conforms to the syntax and semantics defined in section 3.
+A program satisfies the "CSAF producer" conformance profile if the program:
 
-## 5.3 Conformance Clause 2: CSAF producer
+* produces output in the CSAF format, according to the semantics defined in section 3.
+* satisfies those normative requirements in section 3 that are designated as applying to CSAF producers.
```

Comment:
We should add an additional point in regards to the file name.

Reply:
A producer pushing JSON documents per JSON text sequences as pure data (stream) will not need a file name nor will it be able to state one. Regardless of that, it should be a conforming producer. Or did I miss the context of the suggestion?

Reply:
After thinking about your comment, I realized that this might be the wrong place for this suggestion. I'll draft a PR with a detailed suggestion and link it here.